### PR TITLE
New saturation and contrast configs

### PR DESCRIPTION
**Pull request by bebeli555**
This adds new Range configs to saturation and contrast like the brightness config i added in #96
The default value is the same and it converts the old config to the new values.
I tested it multiple times and everything works.
The min and max values are about the same as previously, a bit lower and higher to the point where they still look good.

### CODE REVIEW

**Comment by NataIynn** [comment]
I do miss the original brightness control setting as its easier to change. But I guess if we can resolve the 'too dark & too bright' areas. I guess would resolve that issue.

**Comment by j-meds** [comment]
> I do miss the original brightness control setting as its easier to change. But I guess if we can resolve the 'too dark & too bright' areas. I guess would resolve that issue.

Right, I kinda just want a few options. instead of spending time tweaking all these numbers we are adding. I have a feeling if we keep adding it will have too many options and configs lol. I do know somebody was working on defaults for site locations, so you'd probably just need to configure these settings in there so you can have consistency in places instead of always constantly tweaking these numbers.

**Comment by NataIynn** [comment]
I think a better option for this kind of 'fine' tuning would have the original default set that 117 made, and have a little (edit) button beside each config to allow different configurations. But also having the original drop down for easy access.

**Comment by yungando** [comment]
For someone who is looking for a vanilla+ experience, the default settings pushed with the plugin make the game look a little over saturated with too much contrast so more options to lower it would be welcomed.

>  I kinda just want a few options. instead of spending time tweaking all these numbers we are adding. I have a feeling if we keep adding it will have too many options and configs lol.

The plugin already allows users different scaling options for how they want the game to look - someone uninterested or overwhelmed by changing from the default settings will already have the problem of seeing too many options and configs. If the plan is to make the options easier to users, then I would go with presets, as someone suggested in the Discord. Otherwise, brightness and contrast sliders are already present in most modern game's graphics settings, so it isn't out of place to give the user a similar experience here.

> I do know somebody was working on defaults for site locations, so you'd probably just need to configure these settings in there so you can have consistency in places instead of always constantly tweaking these numbers.

I see the brightness, saturation and contrast setting being the default settings that work alongside specific site location settings, not replacing it. Set the base contrast for you client and then each location modifies the lighting and brightness depending on their individual settings and whether Atmospheric Light is enabled.

Brightness, contrast and saturation are all important settings for accessibility; their inputs being a range like Draw Distance is a good change imo.

**Comment by sosodev** [comment]
@bebeli555 If you rebase this I'd be glad to review it. Sorry it was ignored initially that was before my time here.
